config.js

```diff
@@ -457,3 +457,35 @@ config.mixpanel = {
     'enabled': false,
     'token': 'f3e9fce119d357b745a8dfa36248d632'
 };
+
+/**
+ * `config.zencoder`
+ *
+ * Configuration namespace for Zencoder
+ *
+ * @param  {Boolean}    enabled                     Whether or not Zencoder video encoding should be enabled
+ * @param  {String}     apiKey                      The Zencoder api key
+ * @param  {String}     watermark                   The URL for the watermark logo
+*/
+config.zencoder = {
+    'enabled': false,
+    'apiKey': '',
+    'watermark': ''
+};
+
+/**
+ * `config.storage`
+ *
+ * Configuration namespace for storage
+ *
+ * @param  {String}     amazons3-bucket             The S3 bucket for storing output videos
+ * @param  {String}     amazons3-access-key         The S3 bucket access key
+ * @param  {String}     amazons3-secret-key         The S3 bucket secret key
+ * @param  {String}     amazons3-region             The S3 bucket region
+*/
+config.storage = {
+    'amazons3-bucket': '',
+    'amazons3-region': '',
+    'amazons3-access-key': '',
+    'amazons3-secret-key': ''
+};
```

Comment:
If this storage key only differs from our `files` storage key in that it's used for video previews instead of general user files, then should it be scoped under `config.zencoder`?